docs/source/io_formats/nuclear_data.rst

```diff
@@ -0,0 +1,353 @@
+.. _usersguide_nuclear_data:
+
+========================
+Nuclear Data File Format
+========================
+
+---------------------
+Incident Neutron Data
+---------------------
+
+
+**/<nuclide name>/**
+
+:Attributes: - **Z** (*int*) -- Atomic number
+             - **A** (*int*) -- Mass number
+             - **metastable** (*int*) -- Metastable state
+             - **atomic_weight_ratio** (*double*) -- Mass in units of neutron masses
+             - **temperature** (*double*) -- Temperature in MeV
+             - **n_reaction** (*int*) -- Number of reactions
```

Comment:
I agree with you. The only complication is that iterating over a hitherto unknown number of groups in Fortran is not as straightforward. I'll take a look again at the HDF5 Fortran API and see if I can get rid of these guys.